.eslintignore

```diff
@@ -1,4 +1,5 @@
 build/**
+dist/*
```

Comment:
`dist/` is a directory that contains compiled JS files during the extension build + deploy step.